aim/web/ui/src/pages/RunDetail/RunOverviewTab/GitInfoCard/GitInfoCard.scss

```diff
@@ -0,0 +1,29 @@
+@use 'src/styles/abstracts' as *;
+
+.InfoSection {
+  overflow-y: scroll;
+  .InfoCard {
+    margin-right: toRem(40px);
+    min-width: fit-content;
+    &:last-child {
+      margin-right: 0;
+    }
+    .InfoCardLabel {
+      margin-bottom: toRem(4px);
+    }
+    .Icon__container {
+      color: $pico;
+    }
+    .InfoCardValue {
+      text-align: center;
+      background: $cuddle-10;
+      border: toRem(1px) solid $cuddle-50;
+      box-sizing: border-box;
+      border-radius: toRem(6px);
+      padding: toRem(6px) toRem(6px);
```

Comment:
Let's change space values to our existing variables.
Instead of `toRem(6px) ` use `$space-xxs`